code/modules/shuttle/super_cruise/orbital_map_components/orbital_map.dm

```diff
@@ -0,0 +1,17 @@
+
+/datum/orbital_map
+	//the primary star. Set to be lavaland by default.
+	var/datum/orbital_object/center = null
+	//A list of all bodies
+	var/list/datum/orbital_object/bodies = list()
+
+//Returns a list of gravitationally relevant bodies.
+/datum/orbital_map/proc/get_relevnant_bodies(datum/orbital_object/source)
```

Comment:
> relevnant

Reply:
Once the distance is high enough, the acceleration towards the gravitational body will be less than 0.0001, at which case the gravity calculations are ignored.

Reply:
I was referring to the typo

Reply:
damn it